internal/errcheck/errcheck.go

```diff
@@ -223,6 +223,17 @@ func (v *visitor) ignoreCall(call *ast.CallExpr) bool {
 		}
 	}
 
+	// Ignore standard library methods that promise not to return a non nil error
+	switch v.pkg.Uses[id].String() {
+	case
+		"func (*bytes.Buffer).Write(p []byte) (n int, err error)",
+		"func (*bytes.Buffer).WriteByte(c byte) error",
+		"func (*bytes.Buffer).WriteRune(r rune) (n int, err error)",
+		"func (*bytes.Buffer).WriteString(s string) (n int, err error)":
```

Comment:
I can't tell if the `bytes` there is the package name or full import path.

Is there a chance of this introducing a false positive if I import a package with import path:

```Go
import "company.org/foo/bar/bytes"

func main() {
    buf := new(bytes.Buffer)
    buf.Write(nil) // Should've checked error because company.org/foo/bar/bytes.Buffer can fail.
}
```